python/cudf/setup.py

```diff
@@ -44,7 +44,8 @@
             "../../cpp/include/cudf",
             "../../cpp/include",
             "../../cpp/build/include",
-            "../../thirdparty/cub",
+            "../../cpp/build/_deps/cub-src",
+            "../../cpp/build/_deps/thrust-src",
```

Comment:
Do we still need thrust here?

Reply:
@shwina could you throw in a PR removing both of these? One less thing to handle on the build side 😄

Reply:
Yup -- will do

Reply:
Thrust/CUB headers are transitive dependencies. If we don't put them here we'll just use the ones from the cuda toolkit. Since the intention is to use the newer release of Thrust/CUB, we do need them here.